src/gui/widgets/rich_label.hpp

```diff
@@ -0,0 +1,382 @@
+/*
+	Copyright (C) 2008 - 2024
+	by Mark de Wever <koraq@xs4all.nl>
+	Part of the Battle for Wesnoth Project https://www.wesnoth.org/
+
+	This program is free software; you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation; either version 2 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY.
+
+	See the COPYING file for more details.
+*/
+
+#pragma once
+
+#include "gui/widgets/styled_widget.hpp"
+
+#include "font/standard_colors.hpp"
+#include "gui/core/widget_definition.hpp"
+#include "help/help_impl.hpp"
+#include "serialization/parser.hpp"
+
+#include <iostream>
+#include <string>
+
+namespace gui2
+{
+namespace implementation
+{
+	struct builder_rich_label;
+}
+
+// ------------ WIDGET -----------{
+
+/**
+ * @ingroup GUIWidgetWML
+ *
+ * A rich_label displays text that can be wrapped but no scrollbars are provided.
+ *
+ * A rich_label has two states because rich_labels are often used as visual indication of the state of the widget it rich_labels.
```

Comment:
Looks like overzealous find and replace here.

Reply:
Yes, will fix it.

Reply:
Fixed.